exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -117,12 +117,15 @@ pub fn panic_if_java_options() {
 pub struct JavaServiceFactory;
 
 impl ServiceFactory for JavaServiceFactory {
-    fn command(&mut self, command: &str) -> Option<Box<CommandExtension>> {
-        use exonum::helpers::fabric;
+    fn service_name(&self) -> &str {
+        "JAVA_SERVICE_FACTORY"
```

Comment:
I'm not sure this would be so simple... :upside_down_face:

Reply:
It's a temporary workaround and doesn't related to the main problem 😄

Reply:
Is it supposed to be the name of the service (it seems sensible given this method name), or could it be distinct from it? If the first, why are there two such methods?